lib/modules/browser_console.js

```diff
@@ -26,12 +26,31 @@ class Log {
     }
 }
 
+function stringifyArg(arg) {
+    return arg.executionContext().evaluate(element => {
+        if (typeof element === 'object' && !(element instanceof RegExp)) {
+            element = JSON.stringify(element);
+        }
+        return String(element);
+    }, arg);
+}
+
+async function stringifyLog(log) {
```

Comment:
Just for compatibility reasons, please avoid using async/await on code outside tests